cells/_automation/ciJobs.nix

```diff
@@ -0,0 +1,25 @@
+{
+  inputs,
+  cell,
+}: let
+  inherit (inputs) nixpkgs iohk-nix;
+  inherit (nixpkgs) lib;
+  inherit (inputs.cells.prelude.library) project extra-compilers;
+
+  ciProject = project.appendModule {
+    flake.variants =
```

Comment:
have I ever mentioned how much I like the "just use functions" approach? :D anyway, seems fine